tools/ci_build/github/apple/package_release_tasks.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+# Note: This script is intended to be called from the CocoaPods package release pipeline or a similar context.
+
+from __future__ import annotations
+
+import argparse
+import os
+import shlex
+import subprocess
+from enum import Enum
+from pathlib import Path
+
+
+class Task(Enum):
+    upload_pod_archive = 1
+    update_podspec = 2
+
+
+def _run(command: list[str], **kwargs):
+    print(f"Running command: {shlex.join(command)}", flush=True)
+    kwargs.setdefault("check", True)
+    return subprocess.run(command, **kwargs)  # noqa: PLW1510  # we add 'check' to kwargs if not present
+
+
+def upload_pod_archive(pod_archive_path: Path):
+    env = os.environ.copy()
+    env.update(
+        {
+            # configure azcopy to use managed identity
+            "AZCOPY_AUTO_LOGIN_TYPE": "MSI",
+            "AZCOPY_MSI_CLIENT_ID": "63b63039-6328-442f-954b-5a64d124e5b4",
```

Comment:
Out of interest does this require some other auth to work? e.g. requires az cli login with MS account that has write permissions to the storage if running locally?

Reply:
it requires a managed identity, which is set up in certain build machine pools. I'm not sure how to run this locally.
@snnn would know more about this.

Reply:
You cannot run this piece of code locally, as your machine would not have this managed identity.